crates/chronicle/src/bootstrap/mod.rs

```diff
@@ -345,6 +351,53 @@ where
         .await?;
 
         Ok((ApiResponse::Unit, ret_api))
+    } else if let Some(matches) = matches.subcommand_matches("import") {
+        let (namespace_id, namespace_uuid) = (
+            matches.value_of("namespace-id").unwrap(),
+            matches.value_of("namespace-uuid").unwrap(),
+        );
+        let namespace = NamespaceId::from_external_id(
+            namespace_id,
+            uuid::Uuid::parse_str(namespace_uuid).unwrap(),
+        );
+
+        let contents = if matches.is_present("stdin") {
+            let mut buffer = String::new();
+            let mut stdin = io::stdin();
+            let bytes = stdin.read_to_string(&mut buffer)?;
+            info!("Loading {bytes} bytes of import data from stdin");
```

Comment:
```suggestion
            info!("Loaded {bytes} bytes of import data from stdin");
```
As these are happening *after* the reads, maybe put in the past tense.